students/stefan_lund/Lesson_2/fizz_buzz.py

```diff
@@ -0,0 +1,134 @@
+# python3

```

Comment:
you have two students folders here, one named `stefan_lund` and `StefanLund` please only use one to avoid confusion.